libraries/AP_HAL_ChibiOS/hwdef/DevEBoxH7v2/hwdef-bl.dat

```diff
@@ -29,15 +29,15 @@ PE8 UART7_TX UART7 NODMA
 define BOOTLOADER_DEBUG SD7
 
 # QuadSPI Flash
-PD11 QUADSPI_BK1_IO0 QUADSPI1
-PD12 QUADSPI_BK1_IO1 QUADSPI1
-PE2 QUADSPI_BK1_IO2 QUADSPI1
-PD13 QUADSPI_BK1_IO3 QUADSPI1
-PB6 QUADSPI_BK1_NCS QUADSPI1
-PB2 QUADSPI_CLK QUADSPI1
+PD11 QUADSPI_BK1_IO0 QUADSPI1 SPEED_HIGH
+PD12 QUADSPI_BK1_IO1 QUADSPI1 SPEED_HIGH
+PE2 QUADSPI_BK1_IO2 QUADSPI1 SPEED_HIGH
+PD13 QUADSPI_BK1_IO3 QUADSPI1 SPEED_HIGH
+PB6 QUADSPI_BK1_NCS QUADSPI1 SPEED_HIGH
+PB2 QUADSPI_CLK QUADSPI1 SPEED_HIGH
 
 # IFace  Device Name  Bus      QSPI Mode  Clk Freq  Size (Pow2)  NCS Delay
-QSPIDEV  w25q         QUADSPI1 MODE3      120*MHZ   23           2
+QSPIDEV  w25q         QUADSPI1 MODE3      66*MHZ    23            1
```

Comment:
I think this definitely requires some more investigation.

Reply:
Fixed now - prescaler was wrong